pandas/tests/io/test_stata.py

```diff
@@ -1475,8 +1475,19 @@ def test_out_of_range_double(self):
 
         df.loc[2, "ColumnTooBig"] = np.inf
         msg = (
-            "Column ColumnTooBig has a maximum value of infinity which is outside "
-            "the range supported by Stata"
+            "Column ColumnTooBig has a maximum value of infinity "
+            "or a minimum value of -infinity which is outside "
+            "the range supported by Stata."
+        )
+        with pytest.raises(ValueError, match=msg):
+            with tm.ensure_clean() as path:
+                df.to_stata(path)
+
+        df.loc[2, "ColumnTooBig"] = -np.inf
```

Comment:
Could you split this `inf` testing into a separate test and use `pytest.mark.parameterize`?

Reply:
You mean creating a new test function in the class that only test the two cases for `np.inf` and `-np.inf`, using `pytest.mark.parameterize` to loop over the two values?

Reply:
Exactly, yes.